src/stores/claimables.js

```diff
@@ -0,0 +1,6 @@
+import { writable } from "svelte/store";
+
+export let veEstimate = writable(undefined);
+export let veClaimables = writable(undefined);
+export let dfEstimate = writable(undefined);
+export let dfClaimables = writable(undefined);
```

Comment:
I think we can merge the content from this file with `airdrops.js` and have just one store for all rewards.